src/network/client.rs

```diff
@@ -20,13 +19,10 @@ impl Client {
     ) -> Result<(Self, mpsc::Receiver<Event>, EventLoop)> {
         let (command_sender, command_receiver) = mpsc::channel(1);
         let (event_sender, event_receiver) = mpsc::channel(1);
-        let keypair = Keypair::generate_ed25519();
```

Comment:
will bring back the ability to start with a seed in another pr.

Reply:
Sure, yeah. I'm not sure that it's a critical feature 🤷‍♀️